apps/mull-ui/src/app/components/navigation/sub-nav-bar/sub-nav-bar-header.scss

```
@@ -0,0 +1,29 @@
+@import '../../../../variables.scss';
+
+.header {
+  background-color: $background-color;
+  padding: 0.5rem 0;
+  position: fixed;
+  top: 3.5rem;
```

There is a variable already available for this value.

See `variables.scss`:
```scss
$top-nav-bar-height-mobile: 3.5rem;
$top-nav-bar-height-desktop: 6.25rem;
```

```suggestion
  top: $top-nav-bar-height-mobile
```